module-1/lab-numpy/your-code/main.py

```diff
@@ -82,15 +142,24 @@
 2 x (0, 1) = (0, 2)
 '''
 print(a)
+
+
+# In[18]:
+
+
 print(d)
 
 
+# In[19]:
+
 
 #12. Multiply a and c. Assign the result to e.
 e = np.multiply(a, c)
 print(e)
 
 
+# In[20]:
+
 
 #13. Does e equal to a? Why or why not?
 '''
```

Comment:
With your first definition of a , e is equal to a. Your code is taking the last a execution which is not an array of integers between 0 and 1. Take a look on it and get back to me with your thoughts.